tests/plugin_filter/util.py

```diff
@@ -111,12 +113,13 @@ def update_plugin_filter(tmp_db, client, template_tab: TemplateTab, filter_dict:
     )
     assert response.status_code == 200
     apply_filter_for_tab.apply(args=(template_tab.id,))
-    return (
+    plugins = (
         tmp_db.session.query(TemplateTab)
         .filter(TemplateTab.id == template_tab.id)
         .first()
         .plugins
     )
+    return set(plugin.id for plugin in plugins)
```

Comment:
directly use a set comparison:

```suggestion
    return {plugin.id for plugin in plugins}
```